feature-libs/product-configurator/rulebased/cpq/cpq-configurator-overview-normalizer.ts

```diff
@@ -0,0 +1,165 @@
+import { Injectable } from '@angular/core';
+import { Converter, TranslationService } from '@spartacus/core';
+import { take } from 'rxjs/operators';
+import { Configurator } from '../core/model/configurator.model';
+import { CpqConfiguratorUtilsService } from './cpq-configurator-utils.service';
+import { Cpq } from './cpq.models';
+
+const NO_OPTION_SELECTED = 0;
```

Comment:
For easier extensibility and customizability please consider avoidng private consts, but rather exposing them as `protected` `readonly` fields of the class where it's used, unless it's justified to have it separate. It's justified i.e. when it's used in more files. Then it should be exported in the public API with a thoughtful name.

Reply:
Pick!

Reply:
Done